packages/core/content-manager/admin/src/preview/components/PreviewHeader.tsx

```diff
@@ -156,7 +156,7 @@ const UnstablePreviewHeader = () => {
   const schema = usePreviewContext('PreviewHeader', (state) => state.schema);
   const meta = usePreviewContext('PreviewHeader', (state) => state.meta);
   const plugins = useStrapiApp('PreviewHeader', (state) => state.plugins);
-  const iframeRef = usePreviewContext('PreviewHeader', (state) => state.iframeRef);
+  const onPreview = usePreviewContext('PreviewHeader', (state) => state.onPreview);
```

Comment:
Actually we could just get this from the DocumentContext instead, it doesn't need to live in both places.